src/components/submit-project/components/edit-project-images/EditProjectImages.jsx

```diff
@@ -84,12 +121,15 @@ const EditProjectImages = (props) => {
       ))}
       {/* add image input */}
       <div className={styles.placeholder}>
-        <div className={styles.placeholderItem} onClick={handleInputClick}>
+        {/* your own image */}
+        <div className={styles.placeholderItem} onClick={handleInputClick} role="input">
```

Comment:
Need to handle all the clickable divs. `eslint(jsx-a11y/click-events-have-key-events)`
Either use a Button component via reactstrap, handle the key event, or ignore this eslint issue if you are confident about doing so.

Please handle all the clickable divs.